tests/playwright/src/model/pages/details-page.ts

```diff
@@ -0,0 +1,54 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+
+import { BasePage } from './base-page';
+
+export abstract class DetailsPage extends BasePage {
+  readonly header: Locator;
+  readonly tabs: Locator;
+  readonly tabContent: Locator;
+  readonly controlActions: Locator;
+  readonly closeButton: Locator;
+  readonly backLink: Locator;
+  readonly labelName: Locator;
+  readonly resourceName: string;
+  readonly heading: Locator;
+
+  constructor(page: Page, resourceName: string) {
+    super(page);
+    this.resourceName = resourceName;
+
+    this.tabContent = page.getByRole('region', { name: 'Tab Content' });
+    this.header = page.getByRole('region', { name: 'Header' });
+    this.tabs = page.getByRole('region', { name: 'Tabs' });
+    this.controlActions = this.header.getByRole('group', { name: 'Control Actions' });
+    this.closeButton = this.header.getByRole('button', { name: 'Close' });
```

Comment:
Add new locator for breadcrumb navigation. And then search for back and close within this locator.